osu.Game/Graphics/Cursor/MenuCursor.cs

```diff
@@ -51,12 +51,16 @@ protected override bool OnMouseMove(MouseMoveEvent e)
         {
             if (dragRotationState != DragRotationState.NotDragging)
             {
+                // make the rotation centre point floating.
+                if (Vector2.Distance(positionMouseDown, e.MousePosition) > 60)
+                    positionMouseDown = Interpolation.ValueAt(0.005f, positionMouseDown, e.MousePosition, 0, Clock.ElapsedFrameTime);
+
                 var position = e.MousePosition;
                 float distance = Vector2Extensions.Distance(position, positionMouseDown);
 
                 // don't start rotating until we're moved a minimum distance away from the mouse down location,
                 // else it can have an annoying effect.
-                if (dragRotationState == DragRotationState.DragStarted && distance > 30)
+                if (dragRotationState == DragRotationState.DragStarted && distance > 80)
```

Comment:
I feel the new distance threshold is a bit aggressive. It allows the cursor to not rotate when moving/spinning in small distances, but is also causing the rotation to no longer occur when dragging near elements around:

https://user-images.githubusercontent.com/22781491/175771450-55c5090b-0066-4da4-ae0f-db578095864b.mp4

I think going with 50 is sort of a middle-ground solution (along with updating the mouse down position interpolating logic to something like 30-40), but that depends on whether the rotation is meant to be used for such scenarios, and whether dragging drawables should be a special factor that would switch the cursor to "rotation" mode regardless of distance.

```suggestion
                if (dragRotationState == DragRotationState.DragStarted && distance > 50)
```

Reply:
I intentionally wanted it to invoke less. Will wait for more feedback.

Reply:
I think it feels better this way.

Reply:
I know this is pretty old at this point but I went to make a discussion about this exact change thinking it was a bug and found this PR. I really enjoyed dragging the cursor in circles while I was deciding what map to play, doing it absentmindedly, or for the fun of it. It kinda blows I have to intentionally shake the mouse around a bunch to make it happen now. Can this be undone or just happen when dragging something on a list?

Reply:
It shouldn't take much effort to enter this mode. You don't need to shake or anything. It's just velocity based.